bundler/spec/commands/info_spec.rb

```diff
@@ -19,8 +19,10 @@
 
       install_gemfile <<-G
         source "#{file_uri_for(gem_repo2)}"
+        gem "rack"
```

Comment:
Is this needed? I think `thin` should bring the dependency transitively anyways? One of the points of adding this information is to be able to figure out dependency information for transtitive dependencies that are not present directly in the lockfile, so not specifying `rack` as a top level dependency represents real usage better I think?

Reply:
@deivid-rodriguez yep, you're 100% right.  I just removed that line.